Resources/Locale/en-US/nyanotrasen/abilities/psionic.ftl

```diff
@@ -13,8 +16,8 @@ metapsionic-pulse-power = You detect that {$power} was used nearby.
 action-name-dispel = Dispel
 action-description-dispel = Dispel summoned entities such as familiars or forcewalls.
 
-action-name-mass-sleep = Mass Sleep
-action-description-mass-sleep = Put targets in a small area to sleep.
```

Comment:
mass sleepgon?

Reply:
Mass sleep has been completely replaced with Regenerative Stasis. A single target power that provides some healing for a target in exchange for putting them to sleep. There wasn't really any way to balance Mass Sleep properly without it just being "The Premium Shitter Power", so I came up with this rework after discussing the problem extensively with the systems' original creator.